compositor_pipeline/src/pipeline/output.rs

```diff
@@ -159,34 +161,50 @@ impl Output {
     pub fn frame_sender(&self) -> Option<&Sender<PipelineEvent<Frame>>> {
         match &self {
             Output::Rtp { encoder, .. } => encoder.frame_sender(),
-            Output::EncodedData { encoder } => encoder.frame_sender(),
+            Output::EncodedData { encoder, .. } => encoder.frame_sender(),
```

Comment:
```suggestion
            Output::EncodedData { encoder } => encoder.frame_sender(),
```
the same for other places